lib/workers/branch/schedule.js

```diff
@@ -65,14 +77,15 @@ function isScheduledNow(config) {
     logger.warn(errorMessage);
     return true;
   }
+  const [validTimezone, error] = hasValidTimezone(config.timezone);
+  if (!validTimezone) {
+    logger.warn(error);
+    return true;
+  }
   let now = moment();
   logger.debug(`now=${now.format()}`);
-  // Adjust the time if repo is in a different timezone to renovate
-  if (config.timezone) {
-    // TODO: check for validity manually
-    now = now.tz(config.timezone);
-    logger.debug(`now=${now.format()}`);
-  }
+  now = now.tz(config.timezone);
```

Comment:
This will fail if `config.timezone` is `null` or `undefined`, so it needs to be moved back into an `if`

Reply:
fixed